internal/state/store.go

```diff
@@ -234,12 +258,17 @@ func (store dbStore) save(state State, key []byte) error {
 		state.LastHeightConsensusParamsChanged, state.ConsensusParams, batch); err != nil {
 		return err
 	}
-	if err := batch.Set(key, state.Bytes()); err != nil {
+	stateMarshallTime := time.Now()
+	stateBytes := state.Bytes()
```

Comment:
Why is the byte-conversion taken out of the measurement? Can we add (maybe above line 261) a comment with the reason?

Reply:
Agree with @jmalicevic, if the goal is to measure access time, then any side effects or processing that is not related to the IO should not be considered (or eliminated as much as possible)